### PR TITLE
[Merged by Bors] - Small IntelliJ extension improvements

### PR DESCRIPTION
**Pull request by jamowei**
Fixed some minor things which I found.
Feel free to change anything, if it not fits your needs ;)

### CODE REVIEW

File: intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordListener.kt [hunk 1]
```diff
@@ -116,9 +118,11 @@ class MirrordListener : ExecutionListener {
                         mirrordEnv["MIRRORD_FILE_OPS"] = fileOps.isSelected.toString()
                         mirrordEnv["MIRRORD_AGENT_TCP_STEAL_TRAFFIC"] = stealTraffic.isSelected.toString()
                         mirrordEnv["MIRRORD_EPHEMERAL_CONTAINER"] = ephemeralContainer.isSelected.toString()
-                        mirrordEnv["MIRRORD_REMOTE_DNS"] = remoteDns.isSelected.toString()
                         mirrordEnv["MIRRORD_TCP_OUTGOING"] = tcpOutgoingTraffic.isSelected.toString()
                         mirrordEnv["MIRRORD_UDP_OUTGOING"] = udpOutgoingTraffic.isSelected.toString()
+                        if(tcpOutgoingTraffic.isSelected || udpOutgoingTraffic.isSelected) {
+                            mirrordEnv["MIRRORD_REMOTE_DNS"] = "true"
+                        }
```
Comment (infiniteregrets) [inline intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordListener.kt:125]: we want to keep this as close to the CLI and VSCode extension, where we keep remote DNS as a toggle option, so it'd best to keep it to the user to decide if they want to use this

**Review by infiniteregrets — CHANGES_REQUESTED** [review]
Hey @jamowei, thanks for the improvements! Left a comment

File: intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordListener.kt (hunk 1 above)
Comment (jamowei) [reply intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordListener.kt:125]: Ok, I will change this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,9 +10,14 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
...
 - mirrord-config: New `mirrord-schema.json` file that contains docs and types which should help the user write their mirrord config files. This file has to be manually generated (there is a test to help you remember).
 
+### Fixed
+- IntelliJ: Fix occurring of small namespace selection window and make mirrord dialogs resizable
+- IntelliJ: Fix bug when pressing cancel in mirrord dialog and rerunning the application no mirrord window appears again
+
 ### Changed
 - mirrord-config: Take `schema` feature out of feature flag (now it's always on).
 - mirrord-config: Add docs for the user config types.
+- IntelliJ: Enable "Remote DNS" option automatically when one of "TCP/UDP Outgoing Traffic" option is enabled
```
Comment (infiniteregrets) [inline CHANGELOG.md:20]: can you please remove this

**Review by infiniteregrets — APPROVED** [review]
awesome, looks good. does it happen everytime that cancel and rerun not show the dialog?

File: CHANGELOG.md (hunk 2 above)
Comment (jamowei) [reply CHANGELOG.md]: Oh yeah sorry I forgot...

**Comment by jamowei** [comment]
> awesome, looks good. does it happen everytime that cancel and rerun not show the dialog?

No not always. I found out that it occurs only sometimes in my local tests, and therefore it was important that at least it should go again by turning it on and off.

**Comment by infiniteregrets** [comment]
> > awesome, looks good. does it happen everytime that cancel and rerun not show the dialog?
> 
> No not always. I found out that it occurs only sometimes in my local tests, and therefore it was important that at least it should go again by turning it on and off.

Great! thanks for fixing this (:

**Pull request closed without merging**